assets/scripts/dialogs/SignInButton.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { goPasswordSignIn } from '../app/routing'
+import { useSelector } from 'react-redux'
+
+function SignInButton () {
+  const locale = useSelector((state) => state.locale.locale)
+  const handleClick = () => goPasswordSignIn(locale)
+
+  return <button onClick={handleClick}>Auth0 Sign-In</button>
+}
+
+export default SignInButton
```

Comment:
Actually, this component is never used anywhere. Why is it included?

Reply:
Looks like it's meant to generate the button used [here](https://github.com/streetmix/streetmix/pull/2035/files#diff-94a38e9371b9cf1868f856a69b9730f2be9161cbc83888ebd8308fc0ee56934bR339). I'm going to commit a change that makes this actually happen.